versioned_docs/version-3.0.0-beta/personal-accounts/Login-Connection.md

```diff
@@ -0,0 +1,21 @@
+---
+id: login-connection
+title: Login-connection
+sidebar_label: Login-connection
+slug: /personal-accounts/login-connection
```

Comment:
please update filename to lowercase - `login-connection.md`

Reply:
Addressed